### PR TITLE
Adds FAQ sections and few text modifications in home page and side nav

### PR DESCRIPTION
**Pull request by SaptakS**
These changes only address the changes needed in the representation  of FAQ in documentation. The links and folder structures to work properly, it will depend on the actual changes in docs in openproject repo.

The actual documentation changes should be handled in: https://github.com/opf/openproject/pull/9065/ and https://github.com/opf/openproject/pull/9163/

cc: @oliverguenther

### CODE REVIEW

File: data/default_nav.yaml [hunk 1]
```diff
@@ -7,27 +7,27 @@
...
-  - title: Cloud Edition Guide
-    path: '/cloud-edition-guide'
+  
+  - title: FAQ
+    path: '/faq'
     generate_categories: true
 
-  - title: Enterprise Edition Guide
+  - title: Enterprise Guide
     path: '/enterprise-edition-guide'
```
Comment (ML-OpenP) [inline data/default_nav.yaml:27]: I believe this needs to be changed to `'/enterprise-guide'`

File: data/default_nav.yaml [hunk 2]
```diff
@@ -7,27 +7,27 @@
...
-
-  - title: Cloud Edition Guide
-    path: '/cloud-edition-guide'
+  
+  - title: FAQ
+    path: '/faq'
     generate_categories: true
 
-  - title: Enterprise Edition Guide
+  - title: Enterprise Guide
```
Comment (ML-OpenP) [inline data/default_nav.yaml:26]: `Enterprise guide` (small "g") please

**Review by ML-OpenP — CHANGES_REQUESTED** [review]
Hi @SaptakS 
Thanks a lot!
Apart from the two small things it looks good to me.

**Review by oliverguenther — APPROVED** [review]
Code-wise changes LGTM, but I haven't double checked the changed paths.

**Pull request merged**